### PR TITLE
Clouds on landingpage

### PR DESCRIPTION
**Pull request by jsms90**

### CODE REVIEW

**Comment by des-des** [comment]
@jsms90 pull description!

**Comment by des-des** [comment]
issue num in each commit message!!

File: modal.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 var modal = document.querySelector("#modal");
```
Comment (des-des) [inline modal.js]: hmm sorry I missed this before, all these vars are in global scope! Can we wrap them up in a function?

Comment (jsms90) [inline modal.js]: if I wrap it in an self-invoking function, it doesn't work...am I missing something?

**Comment by SavageWilliam** [comment]
@jsms90 I can't see the new 'goals.html' file anywhere?

**Pull request merged**